src/tasks/withdraw/value.ts

```diff
@@ -72,10 +72,25 @@ export const usdValue = async function (
         network,
       }),
     );
+    // The services return the quote token used for the price. The quote token
+    // is checked to make sure that the returned price meets our expectations.
+    if (
+      response.data.token.toLowerCase() !==
+      REFERENCE_TOKEN[network].address.toLowerCase()
```

Comment:
Although I don't see this constant defined in this PR, I would think its best practice to use `THING.get(key)?` instead of by direct access with `THING[key]` these days.... Just a nit.

Reply:
In general it makes sense, I could use `Map` to obtain a `get` function for free. In this case I'm not so sure because `REFERENCE_TOKEN` is a hardcoded constant, and at least the current notation for creating a structured constant is I believe nicer than using a `Map` or implementing a class with `get`.